pkg/services/ngalert/accesscontrol.go

```diff
@@ -213,6 +219,9 @@ var (
 				{
 					Action: accesscontrol.ActionAlertingProvisioningRead, // organization scope
 				},
+				{
+					Action: accesscontrol.ActionAlertingRulesProvisioningRead, // organization scope
```

Comment:
Q: Is it intentional not to add the Notifications one to either fixed role? Is this how we're intending users to separate the permissions without creating a custom role?

Maybe it's better to have a different fixed role for this given their names/descriptions.

Reply:
Yes, I intentionally y did not add them to the fixed roles due to the following reasons:
1. we're migrating to new APIs that use more granular roles. So, the plan is the same as for rules: control provisioning APIs using the regular API permissions + set provisioning status role.
2. We already have fixed role that gives access to provisioning APIs. Creating 2 more without removing the old one could be confusing for users, I think. Replacing old one role would require some migration, which I want to avoid because it's backported to 10.2